pkg/gohai/go.mod

```diff
@@ -5,22 +5,28 @@ module github.com/DataDog/datadog-agent/pkg/gohai
 go 1.20
 
 require (
-	github.com/cihub/seelog v0.0.0-20151216151435-d2c6e5aa9fbf
+	github.com/DataDog/datadog-agent/pkg/util/log v0.47.0
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought (for a future PR):** ‏we can use the `datadog-agent` log package in `gohai` instead of `seelog` directly